Northstar.Client/mod/scripts/vscripts/ui/panel_mainmenu.nut

```diff
@@ -562,7 +548,7 @@ void function TryAuthWithLocalServer()
 		WaitFrame()
 	}
 
-	if ( NSWasAuthSuccessful() )
+	if ( NSWasAuthSuccessful() || GetConVarBool( "ns_auth_allow_insecure" ) )
```

Comment:
Actually, good change which might work better as a separate PR?

Reply:
> Actually, good change which might work better as a separate PR?

This PR is never getting merged anyway dude I doubt making another one will help